index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>My Project</title>
+</head>
+<body>
+
+    <header>
+        <nav>
+          <div>Awesome Books</div>
+            <ul class="navList">
+              <li><a id="list" href="#">List</a> </li>
+              <li><a id="new" href="#">Add new</a> </li>
+              <li><a id="contact" href="#">Contact</a> </li>
+            </ul>
+        </nav>
+        <p class="time">March 19th 2021, 9:25:32 am</p>
```

Comment:
- [ ] This date is not meant to be static, it should be dynamically created using `luxon`. Any date a user opens your app, it should show the date and the time the user opens your app